src/components/Swap/Status/StatusDialog.tsx

```diff
@@ -34,32 +34,22 @@ function TransactionStatus({ tx, onClose }: TransactionStatusProps) {
     return tx.receipt?.status ? <Trans>Success</Trans> : <Trans>Transaction submitted</Trans>
   }, [tx.receipt?.status])
 
-  const subheading = useMemo(() => {
-    if (tx.receipt?.status) {
-      return <Trans>Your transaction was successful</Trans>
-    }
-    return null
-  }, [tx.receipt?.status])
-
   return (
-    <Column flex padded gap={0.75} align="stretch" style={{ height: '100%' }} data-testid="status-dialog">
+    <Column flex padded align="stretch" style={{ height: '100%', marginTop: '3em' }} data-testid="status-dialog">
       <StatusHeader icon={Icon} iconColor={tx.receipt?.status ? 'success' : undefined}>
-        <ThemedText.H4>{heading}</ThemedText.H4>
+        <ThemedText.H4 margin="3em 0 0">{heading}</ThemedText.H4>
```

Comment:
nit: pref `styled(Component)` wrapping over inline style

Reply:
agree, but `ThemedText.H4` would overwrite it unless you use the prop - this is what the prop is intended for (similar to `Row`)